tests/Events.elm

```diff
@@ -0,0 +1,84 @@
+module Events exposing (..)
+
+import Expect
+import Html exposing (Html, button, div, text, input)
+import Html.Attributes as Attr exposing (href)
+import Html.Events exposing (onClick, onInput)
+import Html.Lazy as Lazy
+import Test exposing (..)
+import Test.Html.Events as Events
+import Test.Html.Query as Query exposing (Single)
+import Test.Html.Selector exposing (tag)
+
+
+all : Test
+all =
+    describe "trigerring events"
+        [ test "returns msg for click on element" <|
+            \() ->
+                Query.fromHtml sampleHtml
+                    |> Query.findAll [ tag "button" ]
+                    |> Query.first
+                    |> Events.msgFor "click" "{}"
+                    |> Expect.equal (Ok SampleMsg)
```

Comment:
Yeah... I've named it `trigger` before it sounds like it is going to do something, like side-effects, when actually it just returns the Msg. Hmmm... but it does execute the parsing of the event, well... I don't know, need more opinions/suggestions please, I'm not happy with `msgFor` either